src/test/java/redis/clients/jedis/BuilderTest.java

```diff
@@ -17,4 +19,17 @@ public void buildDouble() {
     build = BuilderFactory.DOUBLE.build("-inf".getBytes());
     assertEquals(Double.valueOf(Double.NEGATIVE_INFINITY), build);
   }
+
+  @Test
+  public void testBuild() {
+    try {
+      Double build = DOUBLE.build("".getBytes());
```

Comment:
@lacinoire I don't mind adding the test but you can simply as it in previous method.